TutorialApplication/MainViewController.m

```diff
@@ -0,0 +1,137 @@
+//
+//  MainViewController.m
+//  TutorialApplication
+//
+//  Created by Aleksandr Yegorov on 1/20/17.
+//  Copyright © 2017 Aleksandr Yegorov. All rights reserved.
+//
+
+#import "MainViewController.h"
+#import "Employee.h"
+#import "Organization.h"
+#import "DetailViewController.h"
+
+@interface MainViewController ()
+
+@property(strong, nonatomic) Organization *organization;
+
+@end
+
+@implementation MainViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+    Employee *emp1 = [[Employee alloc] initWithName:@"Stepka" lastName:@"Pupkin" salary:3000];
+    
+    self.organization =[[Organization alloc] initWithName:@"TheBestOrganization"];
+    
+    [self.organization addEmployee:emp1];
+    [self.organization addEmployeeWithName:@"Alex Yegorov"];
+    [self.organization addEmployeeWithName:@"Vova Kynovskiy"];
+    [self.organization addEmployeeWithName:@"Misha Davidayn"];
+    
+    NSLog(@"%@", [self.organization description]);
+    
+    NSLog(@"Average salary in the organization = %d", [self.organization calculateAverageSalary]);
+    
+    NSLog(@"%@",[self.organization employeeWithLowestSalary]);
+    
+    NSLog(@"Employees that match the condition: %@", [self.organization employeesWithSalary:2000 tolerance:1000]);
+    
+    [self.organization removeEmployee:emp1];
+    
+    NSLog(@"%@", [self.organization description]);
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+    
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    
+    return [self.organization.employeers count];
+}
+
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"cell" forIndexPath:indexPath];
+    
+    if (cell == nil)
+    {
+        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:@"cell"];
+    }
+    cell.textLabel.text = [self.organization.employeers objectAtIndex:indexPath.row].firstName;
```

Comment:
To access array elements use modern Objective-C syntax, not objectAtIndex. http://clang.llvm.org/docs/ObjectiveCLiterals.html